annotation/tests/test_distribution.py

```diff
@@ -1277,3 +1286,97 @@ def test_get_page_number_combinations(
     expected: List[int],
 ) -> None:
     assert get_page_number_combinations(file_pages, user_pages) == expected
+
+
+@pytest.mark.parametrize(
+    (
+        "split_multipage_doc_setup",
+        "validation_type",
+        "annotators",
+        "validators",
+        "annotation_tasks",
+        "expected_output",
+    ),
+    (
+        (
+            "",
+            ValidationSchema.cross,
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [{"user_id": "2"}],
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [],
+        ),
+        (
+            "1",
+            ValidationSchema.hierarchical,
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [{"user_id": "2"}],
+            [{"user_id": "1"}],
+            [{"user_id": "2"}],
+        ),
+    ),
+)
+def test_choose_validators_users(
+    split_multipage_doc_setup: str,
+    validation_type: ValidationSchema,
+    annotators: List[User],
+    validators: List[User],
+    annotation_tasks: List[Task],
+    expected_output: List[User],
```

Comment:
You pass List of dicts, not User or Task

Reply:
I thought that was okay cause I saw Task type defined as:
Task = Dict[str, Union[int, List[int], bool]]

Reply:
Also, Users are for some reason unsubscriptable and you cant call user['user_id'] on a User object, so I went with dictionaries for Users as thats probably how they are stored in the database